### PR TITLE
Fix relative imports for Python 3

### PR DESCRIPTION
**Pull request by gnossen**
```
Traceback (most recent call last):
  File "/usr/local/google/home/zasweq/workplace/grpc/grpc/tools/interop_matrix/create_matrix_images.py", line 34, in <module>
    import dockerjob
  File "/usr/local/google/home/zasweq/workplace/grpc/grpc/tools/run_tests/python_utils/dockerjob.py", line 25, in <module>
    from . import jobset
ImportError: attempted relative import with no known parent package
```

[Interop Matrix Ad Hoc Manual Run](https://sponge2.corp.google.com/602ed6de-5e73-4b84-a6bd-4a6380666c31) (in-progress)

### CODE REVIEW

File: tools/run_tests/python_utils/dockerjob.py [hunk 1]
```diff
@@ -21,8 +21,11 @@
 import tempfile
 import time
 import uuid
+import sys
 
-from . import jobset
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
+
```
Comment (lidizheng) [inline tools/run_tests/python_utils/dockerjob.py:27]: Extra space? What would YAPF recommend?

**Review by lidizheng — APPROVED** [review]

File: tools/run_tests/python_utils/dockerjob.py (hunk 1 above)
Comment (gnossen) [reply tools/run_tests/python_utils/dockerjob.py]: Looks like yapf just preserves whatever spacing you have in this case. Removed.

**Comment by lidizheng** [comment]
@gnossen 

This PR might be the culprit of 3 failing Windows tests since yesterday:
* [prod:grpc/core/master/windows/grpc_basictests_python](https://fusion.corp.google.com/projectanalysis/summary/KOKORO/prod:grpc%2Fcore%2Fmaster%2Fwindows%2Fgrpc_basictests_python)
* [prod:grpc/core/master/windows/grpc_basictests_csharp](https://fusion.corp.google.com/projectanalysis/summary/KOKORO/prod:grpc%2Fcore%2Fmaster%2Fwindows%2Fgrpc_basictests_csharp)
* [prod:grpc/core/master/windows/grpc_basictests_c](https://fusion.corp.google.com/projectanalysis/summary/KOKORO/prod:grpc%2Fcore%2Fmaster%2Fwindows%2Fgrpc_basictests_c)

The error log looks like:

```
Traceback (most recent call last):
  File "workspace_python_windows_opt_native/tools/run_tests/run_tests.py", line 1897, in <module>
  File "workspace_python_windows_opt_native/tools/run_tests/run_tests.py", line 1823, in _build_and_run
  File "T:\src\github\grpc\workspace_python_windows_opt_native\tools\run_tests\python_utils\jobset.py", line 594, in run
    if not js.start(cmdline):
  File "T:\src\github\grpc\workspace_python_windows_opt_native\tools\run_tests\python_utils\jobset.py", line 479, in start
    self._quiet_success)
  File "T:\src\github\grpc\workspace_python_windows_opt_native\tools\run_tests\python_utils\jobset.py", line 268, in __init__
    self.start()
  File "T:\src\github\grpc\workspace_python_windows_opt_native\tools\run_tests\python_utils\jobset.py", line 317, in start
    self._process = try_start()
  File "T:\src\github\grpc\workspace_python_windows_opt_native\tools\run_tests\python_utils\jobset.py", line 304, in <lambda>
    env=env)
  File "C:\Python37\lib\subprocess.py", line 645, in __init__
    _cleanup()
  File "C:\Python37\lib\subprocess.py", line 215, in _cleanup
    res = inst._internal_poll(_deadstate=sys.maxsize)
  File "C:\Python37\lib\subprocess.py", line 1193, in _internal_poll
    if _WaitForSingleObject(self._handle, 0) == _WAIT_OBJECT_0:
OSError: [WinError 6] The handle is invalid
```

**Comment by lidizheng** [comment]
Link to the WIP fix: https://github.com/grpc/grpc/pull/27983

**Pull request merged**